### ISSUE TITLE
Wrongful errors for formData / type: integer

### ISSUE THREAD

**Issue by netzkind** [CacheControl/hippie-swagger#21]
Affected version: HippieSwagger 2.1.0

Specification:
```
parameters:
  quantity:
    name: quantity
    in: formData
    type: integer
    format: int32
    required: true
```

Code/Request:
```
hippieSwagger(app, dereferencedSwagger)
	.form()
	.post('/url')
	.send({
		quantity: 1
	})
	.expectStatus(200)
```

Result:
>  Error: Invalid format for parameter {quantity}, received: 1.  errors:data should be integer

As far as i have seen, this happens because hippie-swagger decomposes the query-string for validating the parameters - meaning 1 (integer) has changed to "1" (string), thus failing in ajv.

**Comment by netzkind** [comment]
I have created tests in hippie-swagger to prove the problem: https://github.com/netzkind/hippie-swagger/commit/e0b7c6fa1990526f5d3535e881ed4d9a5d31d5d9

I see however no easy fix:
apparantly, the method only receives a querystring-version of the formData-parameters from hippie, and is thus not able to access the original parameters and their type.

**Comment by netzkind** [comment]
Will this issue be addressed?
It's currently a deal-breaker for any specification using formData and numeric parameters.

**Comment by CacheControl** [comment]
Fixed in **3.0.0**
